thunder/core/jit_ext.py

```diff
@@ -915,6 +915,23 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
     return _interpret_call(custom_forward, wrapped_ctx, *args_, **kwargs_)
 
 
+@run_once
+def _warn_custom_torch_info_function():
+    warnings.warn("`torch.finfo` is not implemented in Thunder. Calling `torch.finfo` directly.")
+
+
+@general_jit_lookaside(torch.finfo)
+def _general_jit_torch_finfo_lookaside(obj: Any):
+    _warn_custom_torch_info_function()
```

Comment:
I don't think we need this warning. This is only for autograd.Function not implementing the backward. So we also don't need the `def _warn_custom_torch_info_function():`.